src/CattyTests/Project/ProjectTests.swift

```diff
@@ -223,14 +223,14 @@ class ProjectsTest: XCTestCase {
         object.name = "newObjectName"
         project.objectList.add(object)
 
-        let list = UserVariable(name: "userList", isList: true)
```

Comment:
The `testCopyObjectWithObjectList` test should test the copy functionality of a Brick having a local list. Currently, this is tested for a variable, so I would suggest to remove the variable and make use either of a list Brick or use the list in a Formula.